umpf

```diff
@@ -197,21 +197,21 @@ usage() {
 	  -v, --version <version>    with tag: overwrite version number [default: 1]
 
 	Commands:
-	  diff <commit-ish>          show patches not in any topic branch (not
+	  diff <umpf|commit-ish>     show patches not in any topic branch (not
 	                             upstream) and patches missing locally
-	  show <commit-ish>          show an useries file from an umpf
+	  show [umpf|commit-ish]     show an useries file from an umpf
 	  tig [umpf]                 browse an umpf interactively, showing state of
 	                             local, remote and remote-tracking topic branches
 
-	  tag <commit-ish>           generate a utag from an umerge
-	  format-patch <utag>        generate a useries file and patch stack
+	  tag [umpf]                 generate a utag from an umerge or useries
```

Comment:
`<commit-ish>` is valid as well, e.g. an _utag_ with additional commits on top of it.

Reply:
Included now.